src/app/_components/NewsletterForm.tsx

```diff
@@ -49,12 +79,19 @@ export function NewsletterForm() {
             error={getError(methods.formState.errors, 'email')}
           />
         </div>
-        <div className="flex min-w-44 [&>*:first-child]:flex-1">
+        <div className="min-w-44 flex-shrink-0 flex-grow">
           <Button type="submit" disabled={isSubmitting}>
             {isSubmitting ? 'Subscribing' : 'Subscribe'}
           </Button>
         </div>
       </div>
+      {isOpen && (
+        <NotificationDialog
+          status={notificationStatus}
+          isOpen={isOpen}
+          setIsOpen={setIsOpen}
+        />
+      )}
     </Form>
```

Comment:
This might be out of scope / too much rework. So, let me know what you think 🙂

Wouldn't it be nice NOT to have to include the toast component here, and only call a function to control it?

Something like this:

```ts
try {
    await postSubscription(values.email)
    toast("Successfully subscribed!", { type: "success" })
} catch (error) {
    toast("Something went wrong!", { type: "error" })
    Sentry.captureException(error)
}
```

Or alternatively:

```ts
try {
    await postSubscription(values.email)
    toast.success("Successfully subscribed!")
} catch (error) {
    toast.error("Something went wrong!")
    Sentry.captureException(error)
}
```